tests/unit/utils/test_win_functions.py

```diff
@@ -133,3 +136,24 @@ def test_get_user_groups_error(self):
         with patch("win32net.NetUserGetLocalGroups", side_effect=mock_error):
             with self.assertRaises(WinError):
                 win_functions.get_user_groups("Administrator")
+
+    @skipIf(not salt.utils.platform.is_windows(), "WinDLL only available on Windows")
+    @skipIf(not HAS_WIN32, "Requires pywin32 libraries")
+    def test_get_user_groups_local_pywin_error(self):
```

Comment:
Can you migrate these new tests to pytest, since we are migrating completely to pytest now?

Reply:
Yup! Will do.